sagemaker-training-compiler/tensorflow/multiple_gpu_single_node/vision-transformer.ipynb

```diff
@@ -0,0 +1,640 @@
+{
```

Comment:
<p>Refer to https://github.com/aws/amazon-sagemaker-examples/blob/main/sagemaker-training-compiler/tensorflow/single_gpu_single_node/vision-transformer.ipynb for the waiting section.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3464/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>